codebuild_ECR_push/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "region_name" {
+  type    = string
+  default = "eu-west-2"
+}
+
+variable "deployment_account_id" {
+  description = "The AWS account id where you are deploying to ECR image to."
+  type        = string
+}
+
+variable "deployment_role_name" {
+  description = "The name of the role used to deploy the ECR image."
+  type        = string
+}
+
+variable "codebuild_service_role_name" {
+  description = "The role code build uses to access other AWS services."
+  type        = string
+}
+
+variable "codebuild_image" {
+  description = "The image that CodeBuild will use, including the tag."
+  type        = string
+}
+
+variable "ecr_context" {
+  description = "The absolute path to folder to run docker build from."
+  type        = string
+}
+
+variable "ecr_image_tag" {
+  description = "The name you want to tag the image."
+  type        = string
+}
+
+variable "dockerhub_username" {
```

Comment:
I can't decide whether we should leave this like this or pass in the secrets manager ARN. I know that ties the module down to secrets manager but I think that keeps the password value out of the statefile.

Reply:
For what it's worth, I've changed this so we have to pass in the literal value of the username and password like in #7, partly for the reason you mentioned (didn't want to tie the module to Secrets Manager).

I was hoping to use the `sensitive` parameter for the password variable but that was introduced in Terraform 0.14, and we're still mostly using 0.12... I'm not sure that `sensitive` would prevent the password value from being in the statefile though.